resources/knative-eventing/Chart.yaml

```diff
@@ -1,9 +1,11 @@
-apiVersion: v1
+apiVersion: v2
 name: knative-eventing
 version: 0.12.0
-tillerVersion: ">=2.7.2-0"
 description: Helm chart for installing Knative Eventing
 keywords:
   - knative
   - knative-eventing
   - kyma
+dependencies:
```

Comment:
But then if we use `apiVersion: v1`, `dependencies` won't work hence we need to remove this.

Reply:
For v1, we have to restore the "requirements.yaml" which defines the dependencies for v1. For V2, the dependencies should be moved into the chart, under the new "dependencies". This is an incompatible change in v2.